README.rst

```diff
@@ -40,13 +40,29 @@ Development instructions
 
 Initial setup and installation:
 
-- recommended: create and activate a python 3.5 virtualenv::
+- recommended: create and activate a python 3.8 virtual environment. Using pyenv:
 
-    virtualenv mep -p python3.5
-    source mep/bin/activate
+    # if pyenv is not installed
+    curl https://pyenv.run | bash
+
+    # install 3.8
+    pyenv install 3.8.14
+
+    # activate 3.8
+    pyenv shell 3.8.14    
```

Comment:
Let's use the `.python-version` file here instead of hard-coding the version in the readme. Is there a way for pyenv to pick that up automatically?